src/components/DialogOpenButton.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState, MouseEvent } from "react";
+import { createPortal } from 'react-dom';
+import Dialog from './Dialog';
+import MovieForm from './MovieForm';
+
+interface MovieData {
+  title: string;
+  release_date: string;
+  poster_path: string;
+  vote_average: number;
+  genres: string[];
+  runtime: number;
+  overview: string;
+}
```

Comment:
As it is using in a lot of places, lets move this interface  to types.ts file and import where it is needed

Reply:
@MykhailoSheliahov I will do this in module 5, cause will add some more tsx files and will create common file for them. Need some more experience in ts to plan in this case